disnake/i18n.py

```diff
@@ -63,7 +63,8 @@
 
 
 class Localized(Generic[StringT]):
-    """
+    """|data_class_constructible|
```

Comment:
all data classes are constructable, what is the difference?

Reply:
I personally dont think that user should construct most of the data classes listed on automod page, for example, thats why i made this differentiation

Reply:
It doesn't matter if you think the user should construct them, what matters is if they're dataclasses. They should be in the dataclass category.